@AH-64D Apache Longbow/Addons/fza_ah64_controls/scripting/functions/hellfire/fn_getAttackProfile.sqf

```diff
@@ -0,0 +1,35 @@
+/* ----------------------------------------------------------------------------
+Function: fza_fnc_getAttackProfile
+
+Description:
+	grabs the ace name for the current attack mode.
+
+Parameters:
+    _heli - the helicopter
+
+Returns:
+	Nothing
+
+Examples:
+    _result = [_heli] call fza_fnc_getAttackProfile;
+
+Author:
+	Rosd6(Dryden)
+---------------------------------------------------------------------------- */
+params ["_heli"];
+
+private _trajectory = _heli getvariable "fza_ah64_hellfireTrajectory";
```

Comment:
Rather than having this, could we remove `fza_ah64_hellfireTrajectory`, and instead use `ace_missileguidance_attackProfile`? That way we can get rid of this translation layer.

Reply:
Actually, never mind, that would probably be a bad idea. Maybe we could just set the `ace_missileguidance_attackProfile` variable on the side, or use `useModeForAttackProfile`